spec/models/course/discussion/post_spec.rb

```diff
@@ -41,9 +41,8 @@
         expect(root_post.first).to eq(graph[:root])
 
         root_children = root_post.second
-        expect(root_children).to contain_exactly([graph[:a], [
-                                                   [graph[:b], []]]],
-                                                 [graph[:c], []])
+        expect(root_children).
+          to contain_exactly([graph[:a], [[graph[:b], []]]], [graph[:c], []])
```

Comment:
This is the best I can come up with >_< 

If this is not an improvement in readability we can just ignore this. 

```ruby
let(:root_children_graph) do 
  [graph[:a], [[graph[:b], []]]], [graph[:c], []]
  # [ graph[:a],
  #   [ 
  #     [ graph[:b], [ ] 
  #     ] 
  #   ]
  # ], [ graph[:c], [ ] ]
end
...
  expect(root_children).to contain_exactly(root_children_graph)
```